src/providers/slot/slot.ts

```diff
@@ -77,7 +81,7 @@ export class SlotProvider {
     return Object.keys(slots).slice(0, daysAhead).reduce((acc: {[k: string]: SlotItem[]}, date) => ({
       ...acc,
       [date]: slots[date],
-    }), {});
+    }),                                                  {});
```

Comment:
Is this intentional?

Reply:
Interesting, this is happening because of the tslint rule to align arguments, looks a bit silly tbh, let me fix it